### PR TITLE
Create movie-py-cli.yml

### PR DESCRIPTION
**Pull request by prateekkalra**

### CODE REVIEW

File: _data/projects/movie-py-cli.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ +- movies
+ +- imdb-movies
+ +- imdb-webscrapping
+ +- imdb
+ +upforgrabs:
+ +  name: "Help Wanted"
+ +  link: https://github.com/lucifer069/movie-py-cli/labels/help%20wanted
+
++--- 
+ +desc: "A simple script for fetching movie details right from the terminal"
```
Comment (shiftkey) [inline _data/projects/movie-py-cli.yml]: This looks like the file contents has been duplicated, and contains some `+` characters that are unnecessary.

**Comment by prateekkalra** [comment]
I corrected it , still the checks fail

**Comment by ritwik12** [comment]
@lucifer069 I think you are not getting it. what @shiftkey  meant is that you are having a "+" symbol at the beginning of every line. see any closed PR for reference. refer [this](https://github.com/up-for-grabs/up-for-grabs.net/pull/716/files)
You will get your mistake.

**Comment by shiftkey** [comment]
@lucifer069 thanks!

**Pull request merged**